demisto_sdk/commands/format/update_generic.py

```diff
@@ -268,11 +268,13 @@ def remove_copy_and_dev_suffixes_from_name(self):
         When developer clones playbook/integration/script it will automatically add _copy or _dev suffix.
         """
         if self.verbose:
-            click.echo('Removing _dev and _copy suffixes from name and display tags')
+            click.echo('Removing _dev and _copy suffixes from name, id and display tags')
         if self.data['name']:
             self.data['name'] = self.data.get('name', '').replace('_copy', '').replace('_dev', '')
         if self.data.get('display'):
             self.data['display'] = self.data.get('display', '').replace('_copy', '').replace('_dev', '')
+        if self.data.get('id'):
```

Comment:
@roysagi @orhovy The flow changes the ID to be the same as the name for **new files** but for old files it doesn't change. It might have been missed there or done on purpose. 
@idovandijk raised this issue for when he changed some existing playbooks, the id was left with `_dev`

Reply:
We should only edit the ID of existing playbooks if they are _dev or _copy to remove them, but otherwise we shouldn't touch their ID because some very old playbooks have weird IDs that are different than the names

Reply:
Right so considering that - we should not change old files - this would do that - wouldn't it?

Reply:
@roysagi current implementation (without this PR) is only to change the name and if the file is new, copy the name to the ID.
This PR says that if there is `_dev` or `_copy` in the ID of the file, regardless if it's new or old, remove it.

Reply:
@roysagi I looked through the repo and could not find references for ids that will be changed, so this is valid IMO.